src/plugins/intel_cpu/src/memory_desc/cpu_blocked_memory_desc.cpp

```diff
@@ -112,8 +112,9 @@ bool CpuBlockedMemoryDesc::canComputeMemSizeZeroDims() const {
 }
 
 size_t CpuBlockedMemoryDesc::getCurrentMemSizeImp() const {
-    int64_t e_size = getOffsetPadding() + 1;  // size in bytes (from begin of data to last element)
+    int64_t e_size = getOffsetPadding();  // size in bytes (from begin of data to last element)
     if (!getShape().hasZeroDims()) {
+        e_size += 1;
```

Comment:
As we discussed offline: this behavior should be fixed in separate PR

Reply:
will be done in PR: https://github.com/openvinotoolkit/openvino/pull/9466